h5p-bildetema/src/components/Header/Header.tsx

```diff
@@ -119,9 +119,14 @@ export const Header: React.FC<HeaderProps> = ({
             <ul role="list" className={styles.languages}>
               {favLanguages.map(language => {
                 return (
-                  <li role="listitem">
+                  <li
+                    role="listitem"
+                    key={language.code}
+                    aria-current={
+                      currentLanguageCode === language.code ? "page" : "false"
```

Comment:
If we set `undefined`, I imagine the entire attribute will disappear.

```suggestion
                      currentLanguageCode === language.code ? "page" : undefined
```